__tests__/Port.test.js

```diff
@@ -0,0 +1,32 @@
+/* global describes it expect */
+const Port = require('../src/Port');
+describe('Port', () => {
```

Comment:
As above, dont need the global comment at the top. Also its good practice to leave a clear line between the imports at the top of the file and the start of the describe/test blocks:
```
const Port = require('../src/Port');

describe('Port', () => {
```